docker-compose.yml

```diff
@@ -31,8 +31,9 @@ services:
       - php
     environment:
       SERVER_NAME: ${SERVER_NAME:-localhost, caddy:80}
-      MERCURE_PUBLISHER_JWT_KEY: ${MERCURE_PUBLISHER_JWT_KEY:-!ChangeMe!}
-      MERCURE_SUBSCRIBER_JWT_KEY: ${MERCURE_SUBSCRIBER_JWT_KEY:-!ChangeMe!}
+      MERCURE_URL: ${MERCURE_URL:-http://caddy/.well-known/mercure}
```

Comment:
If going under the `php` service, we can't write `MERCURE_URL: ${MERCURE_URL:-http://caddy/.well-known/mercure}` because in that way, the value from the env var will be fetched from the `.env` file (which is not what we want because of the recipe)